openquakeplatform/openquakeplatform/ript/static/ript/js/examples/101.js

```diff
@@ -0,0 +1,54 @@
+$(document).ready(function () {
+    $('.ex_gid #description').val('The description of exposure function');
+    $('.ex_gid #costStruc').val('per_asset');
+    $('.ex_gid #costStruc').trigger('change');
+    { // blocks are used to reflect DOM hierarchy
+        $('.ex_gid #structural_costs_units').val('GBP');
+        $('.ex_gid #structural_costs_units').trigger('change');
+
+        $('.ex_gid #retroChbx').prop('checked', true);
+        $('.ex_gid #retroChbx').trigger('change');
+        $('.ex_gid #limitSelect').val('relative');
+        $('.ex_gid #limitSelect').trigger('change');
+        $('.ex_gid #deductibleSelect').val('absolute');
+        $('.ex_gid #deductibleSelect').trigger('change');
+    }
+    $('.ex_gid #costNonStruc').val('aggregated');
+    $('.ex_gid #costNonStruc').trigger('change');
+    {
+        $('.ex_gid #nonstructural_costs_units').val('CAD');
+        $('.ex_gid #nonstructural_costs_units').trigger('change');
+    }
+
+    $('.ex_gid #costContent').val('per_area');
+    $('.ex_gid #costContent').trigger('change');
+    {
+        $('.ex_gid #contents_costs_units').val('AUD');
+        $('.ex_gid #contents_costs_units').trigger('change');
+
+        $('.ex_gid #perAreaSelect').val('aggregated');
+        $('.ex_gid #perAreaSelect').trigger('change');
+        $('.ex_gid #area_units').val('Km²');
+        $('.ex_gid #area_units').trigger('change');
+    }
+    $('.ex_gid #costBusiness').val('aggregated');
+    $('.ex_gid #costBusiness').trigger('change');
+    {
+        $('.ex_gid #busi_inter_costs_units').val('NOK/day');
+        $('.ex_gid #busi_inter_costs_units').trigger('change');
+    }
+
+    $('.ex_gid #occupantsCheckBoxes [value="night"]').prop('checked', true);
+    $('.ex_gid #occupantsCheckBoxes [value="night"]').trigger('change');
+
+    var data = [[], []];
+    for (var e = 0 ; e < 2 ; e++) {
```

Comment:
I guess it would be better to give a name to `2` and `14`, to make this more readable